packages/runtime/src/execute/job.ts

```diff
@@ -58,11 +59,17 @@ const calculateNext = (job: CompiledJobNode, result: any) => {
         if (typeof edge.condition === 'function') {
           try {
             if (!edge.condition(result)) {
+              logger.always(
```

Comment:
We shouldn't log this stuff to `always`, it's not important enough. `always` is for critical information that users need to know.

If the system is trusted and working, you know the result of a condition because the next job did or did not fire. This log line imparts no new information - it just confirms that failed edges had their expression tested.

I think we should log this to `debug`. It's the kind of thing you might want to see when something goes wrong and you're trying to trace exactly what happened.

Ditto for the true condition.

Reply:
There's a bit more to think about really.

The condition that gets printed here may not be the same thing the user entered in Lightning. In fact the conditions we generate right now are very far removed from what's in the lightning data and UI. So this may not be useful for end users. Maybe that will get better in time.

Also the condition may not return `true` or `false`, but truthy or falsy values (especially with custom expressions). To the language may be a little confusing.

We can do this in the short term, but in the medium term I think we need to use the condition label (or if not present, use the two job names `job-1 -> job-2`)